lib/Resource/UserWithOrganizationIdResponse.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace WorkOS\Resource;
+
+/**
+ * Class UserWithOrganizationIdResponse.
+ *
+ * @property User $user
+ * @property string $organizationId
+ */
+class UserWithOrganizationIdResponse extends BaseWorkOSResource
```

Comment:
Maybe this can be called AuthenticateResponse? We may want to add to this in the future without having to rename the class.